contrib/internal/httputil/gen/gen.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
Let's document this package and explain what it does. It shouldn't be a package level documentation, it should be the documentation of this utility, something like "This program generates the implementation of all possible combinations of the http.ResponseWriter's interface implementations [...]" or something similar. To make it clear that it's not package level documentation we should add a space between the documentation and package declaration.